#! /bin/sh /usr/share/dpatch/dpatch-run
## 00_debian_default_plugins.dpatch by Christopher James Halse Rogers <raof@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Enable a bunch of useful, official plugins which don't require 
## DP: configuration by default, so gnome-do is more useful out of the box.

@DPATCH@
Index: gnome-do-plugins/Firefox/Resources/Firefox.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/Firefox/Resources/Firefox.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/Firefox/Resources/Firefox.addin.xml.in	2011-02-20 16:59:44.905550591 +1100
@@ -6,7 +6,7 @@
 	description="Search Firefox 3 bookmarks, bookmark directories, and history."
 	author="David Siegel, Neal Stewart, Alex Launi"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/Firefox_Plugin"
 	>
 
Index: gnome-do-plugins/GNOME-Dictionary/Resources/GNOME-Dictionary.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/GNOME-Dictionary/Resources/GNOME-Dictionary.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/GNOME-Dictionary/Resources/GNOME-Dictionary.addin.xml.in	2011-02-20 17:00:13.035591855 +1100
@@ -6,7 +6,7 @@
 	description="Look up word definitions in GNOME Dictionary."
 	author="David Siegel"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/GNOME_Dictionary_Plugin"
 >
  
Index: gnome-do-plugins/GNOME-Screenshot/Resources/GNOME-Screenshot.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/GNOME-Screenshot/Resources/GNOME-Screenshot.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/GNOME-Screenshot/Resources/GNOME-Screenshot.addin.xml.in	2011-02-20 17:00:56.125655048 +1100
@@ -6,7 +6,7 @@
 	description="Take screenshots with optional delay."
 	author="David Siegel"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/GNOME_Screenshot_Plugin"
 >
  
Index: gnome-do-plugins/GNOME-Session/Resources/GNOME-Session.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/GNOME-Session/Resources/GNOME-Session.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/GNOME-Session/Resources/GNOME-Session.addin.xml.in	2011-02-20 17:01:10.955676800 +1100
@@ -6,7 +6,7 @@
 	description="Log out, lock your screen, shutdown, suspend, etc."
 	author="David Siegel"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/GNOME_Session_Plugin"
 >
  
Index: gnome-do-plugins/GNOME-Terminal/Resources/GNOME-Terminal.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/GNOME-Terminal/Resources/GNOME-Terminal.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/GNOME-Terminal/Resources/GNOME-Terminal.addin.xml.in	2011-02-20 17:01:24.175696196 +1100
@@ -6,7 +6,7 @@
 	description="Run commands and open new terminals."
 	author="David Siegel, Tim McLaughlin"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/GNOME_Terminal_Plugin"
 >
  
Index: gnome-do-plugins/GoogleCalculator/Resources/GoogleCalculator.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/GoogleCalculator/Resources/GoogleCalculator.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/GoogleCalculator/Resources/GoogleCalculator.addin.xml.in	2011-02-20 17:01:45.915728081 +1100
@@ -6,7 +6,7 @@
 	description="Perform calculations, and currency/quantity conversions."
 	author="David Siegel"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/Google_Calculator_Plugin"
 >
  
Index: gnome-do-plugins/OpenSearch/Resources/OpenSearch.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/OpenSearch/Resources/OpenSearch.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/OpenSearch/Resources/OpenSearch.addin.xml.in	2011-02-20 17:02:01.995751659 +1100
@@ -6,7 +6,7 @@
 	description="Search the web using your installed OpenSearch plugins."
 	author="Jason Jones"	
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/OpenSearch_Plugin"
 >
  
Index: gnome-do-plugins/Pastebin/Resources/Pastebin.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/Pastebin/Resources/Pastebin.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/Pastebin/Resources/Pastebin.addin.xml.in	2011-02-20 17:02:22.345781507 +1100
@@ -6,7 +6,7 @@
 	description="Paste text to an online pastebin."
 	author="Jason Jones"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/Pastebin_Plugin"
 >
  
Index: gnome-do-plugins/TinyUrl/Resources/TinyUrl.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/TinyUrl/Resources/TinyUrl.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/TinyUrl/Resources/TinyUrl.addin.xml.in	2011-02-20 17:03:29.945880656 +1100
@@ -6,7 +6,7 @@
 	description="Make unwieldy urls tiny."
 	author="David Siegel"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/TinyUrl_Plugin"
 	>
 
Index: gnome-do-plugins/Tomboy/Resources/Tomboy.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/Tomboy/Resources/Tomboy.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/Tomboy/Resources/Tomboy.addin.xml.in	2011-02-20 17:03:49.285909021 +1100
@@ -6,7 +6,7 @@
 	description="Search and create new Tomboy notes."
 	author="Rick Harding, Chris Szikszoy"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/Tomboy_Plugin"
 	>
 
Index: gnome-do-plugins/WindowManager/Resources/WindowManager.addin.xml.in
===================================================================
--- gnome-do-plugins.orig/WindowManager/Resources/WindowManager.addin.xml.in	2011-02-16 21:16:49.000000000 +1100
+++ gnome-do-plugins/WindowManager/Resources/WindowManager.addin.xml.in	2011-02-20 17:04:03.105929297 +1100
@@ -6,7 +6,7 @@
 	description="Control your windows or groups of windows with Do."
 	author="Jason Smith"
 	category="Official"
-	defaultEnabled="false"
+	defaultEnabled="true"
     url="http://do.davebsd.com/wiki/Window_Manager_Plugin"
 	>
 
